3. Java Servlet Web App Tutorial/1-Authentication/sign-in-auth-context/README.md

```diff
@@ -0,0 +1,178 @@
+---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - msal-java
+  - entra
+urlFragment: java-servlet-webapp-authentication
+description: "This sample demonstrates a Java Servlet web app that signs users in with the Microsoft Entra platform and steps up the authentication challenge on-demand."
+---
+# Enable your Java Servlet web app to sign in users to your Azure Active Directory tenant with the Microsoft identity platform
+
+
+## Overview
+
+This sample demonstrates a Java Servlet web app that signs users in to your Microsoft Entra tenant using the [Microsoft Authentication Library (MSAL) for Java](https://learn.microsoft.com/entra/msal/java/).
+It also demonstrates how to step-up the authentication challenge by requiring multi-factor authentication (MFA) for certain routes.
+
+![Overview](./ReadmeFiles/topology.png)
+
+>**Note**
+>Make sure you understand the [basic web site authentication sample](https://github.com/Azure-Samples/ms-identity-msal-java-samples/tree/bogavril/auth-context/3-java-servlet-web-app/1-Authentication/sign-in) first. This shows how to sign users in.
```

Comment:
I updated the link here to fix the broken path

Reply:
I'll make sure my next change follows the proper format. Suggested changes like Den. My bad